app/controllers/idv/confirmations_controller.rb

```diff
@@ -22,8 +22,15 @@ def submit_answers
       agent = Proofer::Agent.new(vendor: idv_vendor, applicant: idv_applicant)
       @idv_vendor = idv_vendor
       @confirmation = agent.submit_answers(idv_resolution.questions, idv_resolution.session_id)
-      complete_idv_profile if @confirmation.success?
+      successful_confirmation if @confirmation.success?
       clear_idv_session
     end
+
+    def successful_confirmation
```

Comment:
@sbc100 that's a good point. I've tried to consistently use the following terms with these definitions in mind:

* proofing - the entire process of verifying user profile assertions
* resolution - 1st part of proofing. Reconciling assertions to a single record
* confirmation - 2nd part of proofing. Verifying that the resolved record belongs to the user.

I will admit to sometimes using "verification" and "confirmation" interchange-ably. I tried to keep the language consistent with how vendors use it, though they are not consistent either.

The `Proofer` class is `Proofer::Confirmation` - https://github.com/18F/identity-proofer-gem/blob/master/lib/proofer/confirmation.rb